### ISSUE TITLE
Ability to pass arguments to application

### ISSUE THREAD

**Issue by brucelh** [DonJayamanne/javaVSCode#21]
Does there currently exist a way to pass command line arguments to the java code that is being debugged? I looked at all the available launch options and didn't see anything in there.

Thanks,
Bruce

**Comment by faustinoaq** [comment]
Because the extension is open source, for now, You can edit the `jdb.js` in the extension folder at the line 105:

```js
        var jdbPath = this.args.jdkPath ? path.join(this.args.jdkPath, "jdb") : "jdb";
        var args = ["-connect", ("com.sun.jdi.SocketAttach:hostname=localhost,port=" + port)];
        this.jdbProc = child_process.spawn(jdbPath, args, {
            cwd: this.args.cwd
        });
```

Then reload VSCode :smiley:

**Comment by faustinoaq** [comment]
Solved via #55
